src/js/nettest.js

```diff
@@ -66,10 +66,9 @@ NetworkTest.prototype = {
           newUrls.push(uri);
         } else if (
           uri.indexOf('?transport=') === -1 && uri.startsWith('turn')) {
-          newUrls.push(uri + '?' + transport);
+            newUrls.push(uri + '?' + transport);
         }
       }
-      iceServer.urls = newUrls;
```

Comment:
If the config contained multiple ice servers each with a list of urls and one of them would get filtered to have zero urls then it would sort of generate an invalid ice server.

So I think you are looking for:

```
  var newIceServers = [];
  for (var i = ...) {
    for (var j = ...) {
    }
    if (newUrls.length !== 0) {
       iceServer.urls = newUrls;
       newIceServers.push(iceServer);
    }
  }
  config.iceServers = newIceServers;

```